soc/arm/microchip_mec/mec172x/linker.ld

```diff
@@ -0,0 +1,9 @@
+/* linker.ld - Linker command/script file */
+
+/*
+ * Copyright (c) 2014 Wind River Systems, Inc.
```

Comment:
Is this correct? Do you want to update it?

Reply:
Done